src/main/java/hudson/plugins/deploy/tomcat/TomcatAdapter.java

```diff
@@ -81,4 +92,15 @@ public void configure(Configuration config, EnvVars envVars, VariableResolver<St
     protected WAR createWAR(File deployableFile) {
         return new TomcatWAR(deployableFile.getAbsolutePath());
     }
+
+    @Override
+    public void redeployFile(
+        FilePath war, String aContextPath, Run<?, ?> run, Launcher launcher, TaskListener listener)
+        throws IOException, InterruptedException {
+        String finalContextPath = aContextPath;
+        if(this.alternativeDeploymentContext != null) {
```

Comment:
Please use ternary if.

Reply:
I would like to use `StringUtils.defaultIfBlank`